app/models/community_resource.rb

```diff
@@ -1,5 +1,8 @@
 class CommunityResource < ApplicationRecord
   taggable_array :tags
+  extend Mobility
```

Comment:
Ok to put this below taggable_array declaration?

Reply:
I think we're removing `taggable_array` soon. I do think there's a soft norm of `extend` and `include` coming first in a class declaration before anything else?

Reply:
yup re removing. yeah, i thought that too re extend and include, but wasn't sure if y'all had preferences re the taggable_array since it is a kind of inclusion.